recipe/emailpassword/utils.go

```diff
@@ -50,6 +50,7 @@ func validateAndNormaliseUserInput(recipeInstance *Recipe, appInfo supertokens.N
 	typeNormalisedInput.GetEmailDeliveryConfig = func(recipeImpl epmodels.RecipeInterface) emaildelivery.TypeInputWithService {
 		sendPasswordResetEmail := defaultCreateAndSendCustomPasswordResetEmail(appInfo)
 		if config != nil && config.ResetPasswordUsingTokenFeature != nil && config.ResetPasswordUsingTokenFeature.CreateAndSendCustomEmail != nil {
+			supertokens.LogWarningMessage("CreateAndSendCustomEmail is deprecated. Use EmailDelivery instead.")
```

Comment:
This is not the right way to mark as deprectaed

You need to do how it's done here: https://github.com/supertokens/supertokens-golang/blob/emailpassword-email-service/recipe/emailverification/evmodels/models.go#L26